packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -40,13 +50,16 @@ contract SystemConfig is OwnableUpgradeable, Semver {
         address _owner,
         uint256 _overhead,
         uint256 _scalar,
-        bytes32 _batcherHash
+        bytes32 _batcherHash,
+        uint64 _gasLimit
     ) public initializer {
+        require(_gasLimit >= MINIMUM_GAS_LIMIT, "SystemConfig: gas limit too low");
```

Comment:
Wondering if we should just call the methods with each value passed in to remove this require check

Reply:
Not sure what's better here. Calling the methods would also emit log events, which I think is a bad pattern during constructors, since the events are not emitted when embedding the contract in a state, so we can't rely on that.